apps/web/src/components/Messages/Composer.tsx

```diff
@@ -111,8 +114,8 @@ const Composer: FC<ComposerProps> = ({
       );
 
       const uploadedAttachment = await uploadFileToIPFS(file);
-      const cid = sanitizeDStorageUrl(uploadedAttachment?.original.url);
-      const url = `${IPFS_GATEWAY}${cid}`;
+      const hash = uploadedAttachment?.original.url.replace('ipfs://', '');
```

Comment:
Do we need to replace?

Reply:
Yes there is currently a bug where we are double prefixing which causes an endless spinner when loading remote attachments!

Reply:
@bigint here's what the url looks like in production currently unfortunately. We didn't catch it at first because cached attachments look great. So only remote attachments are failing with 404 & bad url. 

<img width="826" alt="Screenshot 2023-05-18 at 10 11 54 AM" src="https://github.com/lensterxyz/lenster/assets/556051/ce2682f8-c362-4b63-a11b-96a6f3d105a3">

Reply:
I guess while storing you guys push with the gateway 👀 ? no?

Id suggest always push the `ipfs://` thing instead with gateway! cuz gateways are not stable and meant to change 🙇🏼

Reply:
Ahhh I see your point! I don't the stripping step is necessary since the sanitize also does that and appends the gateway. I just tested without this line and it works. We just need to remove prepending the gateway here and let the sanitize method do that. I updated to account for that!

Reply:
amazing